solr/core/src/java/org/apache/solr/handler/admin/CoreAdminOperation.java

```diff
@@ -380,8 +380,14 @@ public static NamedList<Object> getCoreStatus(
                 SimpleOrderedMap<Object> indexInfo =
                     LukeRequestHandler.getIndexInfo(searcher.get().getIndexReader());
                 long size = core.getIndexSize();
+                long onDiskSize = core.getOnDiskSize();
+                String readableSize = NumberUtils.readableSize(size);
+                // avoid re-parsing size if we don't need to, since it's a somewhat expensive operation
```

Comment:
Code's fine as-is, but this comment is misleading and should be removed I think (pretty sure `NumberUtils.readableSize()` is not expensive).

Reply:
Ah, what were you referring to earlier with this comment? https://github.com/cowpaths/fullstory-solr/pull/212#discussion_r1723474012